src/test/java/uk/gov/pay/api/json/CreatePaymentRequestDeserializerTest.java

```diff
@@ -509,6 +553,54 @@ public void deserialize_shouldThrowValidationException_whenLanguageIsEmptyString
         deserializer.deserialize(jsonFactory.createParser(json), ctx);
     }
 
+    @Test
+    public void deserialize_shouldThrowValidationException_whenDelayedCaptureIsAString() throws Exception {
```

Comment:
Minor, but I think `IsNotABoolean` (rather than `IsAString`) might capture the intent of this test better